### PR TITLE
Feature/w3a-10 Twitter User Credibility

### PR DESCRIPTION
**Pull request by martinezfabiola**

### CODE REVIEW

File: src/calculator/routes.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+calculatorRoutes.get('/plain-text', function(req, res) {
+  res.send(textCredibility(req.query.text, {
+    weightBadWords: +req.query.weightBadWords,
+    weightMisspelling: +req.query.weightMisspelling,
+    weightSpam: +req.query.weightSpam
+  }))
+})
+
+calculatorRoutes.get('/twitter/social/:userId', function(req, res, next) {
```
Comment (davideluque) [inline src/calculator/routes.ts]: Here, it should be /twitter/user/:userId, not social

File: .eslintrc.yml [hunk 2]
```diff
@@ -14,6 +14,10 @@ parserOptions:
 plugins:
   - '@typescript-eslint'
 rules:
+  no-unused-vars:
+    - off
```
Comment (davideluque) [inline .eslintrc.yml]: What is the point of turning this of? just for curiosity

Comment (german1608) [reply .eslintrc.yml]: That rule is for javascript projects, and conflicts with the typescript-eslint plugin rule. I disable it to use the typescript one

File: src/calculator/routes.ts (hunk 1 above)
Comment (martinezfabiola) [reply src/calculator/routes.ts]: Because you change my code

Comment (davideluque) [reply src/calculator/routes.ts]: La loca @martinezfabiola

**Pull request merged**